alert_test.go

```diff
@@ -95,6 +95,59 @@ func TestClient_CreateAlertRule(t *testing.T) {
 	assert.Equal(t, &expected, res)
 }
 
+func TestClient_CreateAlertRuleWithNotifications(t *testing.T) {
+	setup()
+	out := `{"alertRuleId": 1, "ruleName": "test", "notifications":{"thirdParty":[{"integrationId": "1", "integrationName": "foo", "integrationType":"bar"},{"integrationId": "2", "integrationName": "foo2", "integrationType":"bar2"}]}}`
+	mux.HandleFunc("/alert-rules/new.json", func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, "POST", r.Method)
+		w.WriteHeader(http.StatusCreated)
+		_, _ = w.Write([]byte(out))
+	})
+
+	var client = &Client{APIEndpoint: server.URL, AuthToken: "foo"}
+	u := AlertRule{
+		RuleID:   Int64(1),
```

Comment:
```suggestion
```
Small nitpick here, the ThousandEyes API returns the value for `RuleID`, so in reality this field should not be populated here, but it should be populated in the `expected` variable, which you did. So this just needs to be removed.

Reply:
fixed the nit and squashed the commits @pedro-te. Thank you for reviewing this promptly. Once this is done I'll make a follow up contribution to your provider repo bumping the version of the sdk. 

Cheers! 🍻